tests/src/test/java/com/maxio/advancedbilling/utils/TestSetup.java

```diff
@@ -418,7 +424,30 @@ public Invoice createInvoice(int subscriptionId, Consumer<CreateInvoice.Builder>
                 .getInvoice();
     }
 
-    public SubscriptionGroupSignupResponse signupWithSubscriptionGroup(Product product, Component component) throws ApiException, IOException {
+    public Invoice createOpenInvoice(int subscriptionId, int productId) throws IOException, ApiException {
```

Comment:
There is already existing createInvoice which can be customized and reused.

Reply:
It is customized and reused 👍